### PR TITLE
Write an abstract for a vegetables presentation

### PR DESCRIPTION
**Pull request by everythingfunctional**
This is a talk about testing strategies and how I use vegetables that I'd like to give at FortranCon. I'm curious about opinions on the abstract.

### CODE REVIEW

File: FortranCon2021-vegetables/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+**Abstract**:
+Automated testing is a well established best practice in software development,
+so you are writing test suites for your Fortran projects right?
+But how useful is your test suite after you have finished writing it?
+Does it help you to pinpoint the source of bugs?
+Does it help you to make changes to functionality, or get in the way?
+Would a new developer find it useful for getting their bearings in your project?
+Can you tell which requirement a given test is meant to verify?
+Experience in the nuclear power industry,
+with it's emphasis on quality assurance and high standards for verification,
```
Comment (awvwgk) [inline FortranCon2021-vegetables/README.md]:
```suggestion
with its emphasis on quality assurance and high standards for verification,
```

**Comment by everythingfunctional** [comment]
This was accepted, so should this be merged now, or wait until I give the presentation and add the slides?

**Comment by milancurcic** [comment]
I'd say merge now, and add slides in another PR.

**Comment by everythingfunctional** [comment]
I don't have permissions to merge, so can someone either give me permission or merge for me? Thanks.

**Comment by milancurcic** [comment]
I just added the @fortran-lang/push-access team to this repo so you should be able to merge now.

**Comment by everythingfunctional** [comment]
Thanks @milancurcic .

**Pull request merged**